ui/components/multichain/network-list-menu/add-rpc-url-modal/add-rpc-url-modal.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+  Box,
+  ButtonPrimary,
+  ButtonPrimarySize,
+  FormTextField,
+} from '../../../component-library';
+import {
+  BlockSize,
+  Display,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+const AddRpcUrlModal = () => {
+  const t = useI18nContext();
+
+  return (
+    <Box padding={4}>
+      <FormTextField
+        id="additional-rpc-url"
+        label={t('additionalRpcUrl')}
+        labelProps={{
+          children: undefined,
+          variant: TextVariant.bodySmMedium,
+        }}
+      />
+
+      <ButtonPrimary
+        size={ButtonPrimarySize.Lg}
+        display={Display.Block}
+        width={BlockSize.Full}
+        marginTop={8}
+        marginLeft={'auto'}
+        marginRight={'auto'}
+      >
+        {t('addUrl')}
+      </ButtonPrimary>
+    </Box>
```

Comment:
if we decide to include input validation here, it would be a good idea to write unit tests for this modal with the different scenarios ( `rpcUrl` doesn't match with chainID, `rpcURL` already added ...etc )